frontend/fingertips-frontend/components/molecules/Search/result/searchResult.test.tsx

```diff
@@ -92,6 +95,13 @@ describe('Search Result Suite', () => {
     );
   });
 
+  it('should have a direct link to the indicator chart', () => {
+    const expectedPath = `/chart?si=test&is=${MOCK_DATA[0].id.toString()}`;
```

Comment:
If you can use the SearchParam enum for the queryParam in here i.e 

```
const expectedPath = `/chart?${SearchParams.SearchedIndicaor}=test&${SearchParams.IndicatorSelected}=${MOCK_DATA[0].id.toString()}`;
```

This is so if we ever change the param names again. We don't have to change it in loads of places.